### PR TITLE
use standard disruption checker for load balancer

### PR DESCRIPTION
**Pull request by deads2k**
replaces https://github.com/openshift/origin/pull/26675, built on https://github.com/openshift/origin/pull/26682

### CODE REVIEW

**Comment by deads2k** [comment]
/payload 4.10 ci blocking
/payload 4.10 nightly blocking

/hold

holding so we can see the payload results.  I think this will actually reduce the downtime observed for service LB because it's more likely to end up as one second chunks.

File: pkg/monitor/backenddisruption/disruption_backend_sampler.go [hunk 1]
```diff
@@ -198,6 +198,10 @@ func (b *BackendSampler) bodyMatches(body []byte) error {
 	return nil
 }
 
+func (b *BackendSampler) SetHost(host string) {
+	b.host = host
+}
```
Comment (dgoodwin) [inline pkg/monitor/backenddisruption/disruption_backend_sampler.go:203]: Is breaking the "With" pattern above intentional due to the late binding nature of it?

**Review by dgoodwin — COMMENTED** [review]
Looks fine to me otherwise.

/lgtm

File: pkg/monitor/backenddisruption/disruption_backend_sampler.go (hunk 1 above)
Comment (deads2k) [reply pkg/monitor/backenddisruption/disruption_backend_sampler.go:203]: > Is breaking the "With" pattern above intentional due to the late binding nature of it?

yes

**Comment by deads2k** [comment]
4.9 doesn't honor the scc label.  I'll have to tweak the namespace too.  Oh teh ugly.

**Comment by deads2k** [comment]
no conflict with previous.  merging.

**Pull request merged**